### DISCUSSION TITLE
Certification (MPAA Ratings) Filters

### DISCUSSION THREAD

**Discussion by nosajthenitram** [sct/overseerr#2154] [Feature Requests]
I'm working on a feature that will synchronize the movie and TV content ratings with Plex to limit the display on the Discover pages of Overseer. The major issue that I see is that a lot of the 'front page' Discover content is not easily filtered due to a lack of the functionality on the TMDB API. 

The question is, would the devs be opposed to hiding some of the MediaSliders when a user's content ratings are < R? This does make for a much less impressive Discover page, but for tweens / teens, this would focus the content in the event that they want to Discover some new media. 

Thoughts?

**Comment by TheCatLady** [comment]
Probably best to join us on Discord to discuss.
